thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/bao/jdbc/AbstractManagerImpl.java

```diff
@@ -97,6 +100,15 @@ public int deleteByPredicate(Predicate predicate) {
     return genericPojoDao.deleteByPredicate(predicate, beanClass);
   }
 
+  @Override
+  @Transactional
```

Comment:
I am not sure too. This annotation was used in the old methods that scatter around different beanClasses of managerImpl. To prevent from breaking anything in the system, let's leave it as is for now.